dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/JsonFieldAdder.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.logging.data;
+
+import com.ibm.ws.logging.data.JSONObject.JSONObjectBuilder;
+
+/**
+ *
+ */
+public interface JsonFieldAdder {
```

Comment:
I don't think it can be moved to AccessLogSource because AccessLogDataFormatter uses the populate method for the field adders. The http transport bundle (where AccessLogSource is located) is not visible to the logging bundle, so we would need to move both